<!-- Parameters for the caros_universalrobot node -->
<launch>
  <!-- Define the parameterisation of the ROS parameters -->
  <arg name="device_name" default="UR-6-85-5-A" />
  <arg name="device_ip" default="127.0.0.1" />
  <arg name="callback_port" default="33333" />
  <arg name="callback_ip" default="127.0.0.1" />

  <!-- Set the ROS parameters -->
  <param name="device_name" type="str" value="$(arg device_name)" />
  <param name="device_ip" type="str" value="$(arg device_ip)"/>
  <param name="callback_port" type="str" value="$(arg callback_port)"/>
  <param name="callback_ip" type="str" value="$(arg callback_ip)"/>
</launch>
